### DISCUSSION TITLE
카카오맵 진행시 질문

### DISCUSSION THREAD

**Discussion by ghost** [changdoc/naver-vaccine-macro#192] [Q&A]
카카오맵을 새로고침하여 발생시에만 네 이버로 시도하는 걸로 알고있는데요..
그렇다면 크롬에서 사용자계정을 여러개해서 동일 아이디로 켜놔도 무방할까요?

**Comment by changdoc** [comment]
동작 방식이 네이버 맵에서 graphql 결과로 생성한 병원 목록으로 네이버 예약 신청 페이지를 여는 거라 동일한 지역 내에서만 탐색이 가능한 상황이라 
제 판단으로는 여러 창을 띄우는건 의미가 없을 것 같습니다.
